src/archive.rs

```diff
@@ -332,6 +338,25 @@ impl<'a> EntriesFields<'a> {
         }
     }
 
+    fn get_pax_extension_size(&mut self, pax: &[u8]) -> Option<u64> {
+        for extension in pax_extensions(pax) {
+            let current_extension = extension.unwrap();
```

Comment:
The `.unwrap()` here should be avoided, this should likely just break out of the loop